src/datasets/datasets.service.ts

```diff
@@ -79,6 +278,41 @@ export class DatasetsService {
     return datasets;
   }
 
+  async findAllComplete(
+    filter: FilterQuery<DatasetDocument>,
+  ): Promise<OutputDatasetDto[]> {
+    const whereFilter: FilterQuery<DatasetDocument> = filter.where ?? {};
+    const fieldsProjection: string[] = filter.fields ?? {};
+    const limits: QueryOptions<DatasetDocument> = filter.limits ?? {
+      limit: 10,
+      skip: 0,
+      sort: undefined,
```

Comment:
It's probably more practical to have `createAt` as dscending by default instead of undefined